### PR TITLE
Create add credentials  page visual components and save value

### PR DESCRIPTION
**Pull request by ajaypratap003**
### Type of change

- Enhancement / new feature
- Refactoring

### Description
1. Fixed label and value issue in case different in DropdownWithToggle component
2. Create ToggleIcon component
3. Create AddCredential component 
4. Create CredentialList component
5. Create SecretList component
6. Create ExtensionList component
7. Save fields value in json object.
8. Create storybook test component 

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/src/modules/device/components/AddCredential/AddCredential.tsx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  }, [type]);
+
+  const onSelectType = (id: string, event: any, value: string) => {
+    setType(value);
+    setActiveCredentialFormId(id);
+    handleInputChange(id, event, value);
+  };
+
+  const addMoreItem = (id?: string, property?: string) => {
+    let newCredentials: any[];
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/device/components/AddCredential/AddCredential.tsx:113]: Can this be a type of ICredential[]?

Comment (ajaypratap003) [reply console/console-init/ui/src/modules/device/components/AddCredential/AddCredential.tsx:113]: I tried to use ICredential[] but some index issue/error was coming that's why i used any[]. I'll take a look when I will work on container part.

File: console/console-init/ui/src/modules/device/components/AddCredential/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from "./AddCredential";
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/modules/device/components/AddCredential/index.ts:1]: License is missing.

File: console/console-init/ui/src/stories/AddCredential.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/stories/AddCredential.stories.tsx:1]: License is missing.

File: console/console-init/ui/src/components/DropdownWithToggle/DropdownWithToggle.tsx [hunk 4]
```diff
@@ -34,16 +35,19 @@ export interface IDropdownWithToggleProps
...
   component?: React.ReactNode;
   toggleClass?: string;
-  value?: string;
-  onSelectItem?: (value: string) => void;
+  value: string;
+  onSelectItem?: (value: string, event?: any) => void;
   dropdownItemIdPrefix?: string;
   dropdownItemClass?: string;
   isDisplayLabelAndValue?: boolean;
+  isLabelAndValueNotSame?: boolean;
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/components/DropdownWithToggle/DropdownWithToggle.tsx:43]: Just one grammatical suggestion I  would like to give: 

`isLabelAndValueNotSame` -> `areLabelAndValueNotSame` 
`isDisplayLabelAndValue` -> `shouldDisplayLabelAndValue`

What do you feel?

Comment (ajaypratap003) [reply console/console-init/ui/src/components/DropdownWithToggle/DropdownWithToggle.tsx:43]: Agreed on second point but not sure about first, i never seen boolean property start with prefix are. i always seen prefix "is". Lets keep it as it as.
Below points sounds good.
isDisplayLabelAndValue -> shouldDisplayLabelAndValue

File: console/console-init/ui/src/modules/device/components/AddCredential/index.ts (hunk 2 above)
Comment (ajaypratap003) [reply console/console-init/ui/src/modules/device/components/AddCredential/index.ts:1]: done

File: console/console-init/ui/src/stories/AddCredential.stories.tsx (hunk 3 above)
Comment (ajaypratap003) [reply console/console-init/ui/src/stories/AddCredential.stories.tsx:1]: done.

**Pull request merged**